chime/repo_functions.py

```diff
@@ -531,6 +535,39 @@ def save_working_file(clone, path, message, base_sha, default_branch_name):
 
     return clone.active_branch.commit
 
+def get_conflict(clone, other_branch_name):
+    ''' Attempt to merge from origin default branch, return a conflict (if any).
+    
+        Currently requires a clean working tree in the clone!
+    '''
+    clone.git.fetch('origin', other_branch_name)
+
+    try:
+        # Try a no-commit merge from the other branch.
+        clone.git.merge(_origin(other_branch_name), '--no-commit')
+
+    except GitCommandError:
+        # Okay, we have a conflict. Make a new MergeConflict and return it.
+        clone.git.merge('--abort')
+        
+        remote_commit = clone.refs[_origin(other_branch_name)].commit
+        return MergeConflict(remote_commit, local_commit=clone.commit())
+    
+    else:
+        # Merged clean, we're clean, everybody's clean.
+        clone.git.reset('--hard')
+
+def get_changed(clone, other_branch_name):
+    ''' Check differenace against origin default branch, return a boolean True if any.
+    '''
+    clone.git.fetch('origin', other_branch_name)
```

Comment:
Use merge-base here? @tmaybe thinks there might be an existing thing in repo_functions for this. Get branch start point maybe.